region/norfair/crocomire.json

```diff
@@ -728,6 +1081,28 @@
                     }}
                   ],
                   "note": "Use the moving platform (Kamer) to elevate the Gamets."
+                },
+                {
+                  "name": "In-room Speedy Jump up with HiJump and Leave while Shinecharged",
+                  "notable": false,
+                  "requires": [
+                    "HiJump",
+                    "canShinechargeMovementComplex",
+                    {"or": [
+                      "canWalljump",
+                      "canTrickyDashJump"
+                    ]},
+                    {"canShineCharge": {
```

Comment:
or 19 tiles with the door open

Reply:
The worry is that you would slide into the transition with an open door.  But it does still provide the extra tile.

Reply:
Makes sense. Feel free to not include it.